mrmustard/math/tensorflow.py

```diff
@@ -292,9 +294,8 @@ def zeros_like(self, array: tf.Tensor) -> tf.Tensor:
     # Special functions
     # ~~~~~~~~~~~~~~~~~
 
-    def DefaultEuclideanOptimizer(
-        self,
-    ) -> tf.keras.optimizers.Optimizer:  # TODO: a wrapper class is better?
+    @staticmethod
+    def DefaultEuclideanOptimizer() -> tf.keras.optimizers.Optimizer:  # TODO: a wrapper class is better?
```

Comment:
(very minor) just to avoid >100 lines. 
```suggestion
    # TODO: is a wrapper class better?
    @staticmethod
    def DefaultEuclideanOptimizer() -> tf.keras.optimizers.Optimizer:
```